src/components/account/settings/General.vue

```diff
@@ -154,8 +154,15 @@ export default Vue.extend({
           time: 5000,
         });
       } catch (e) {
+        /**
+         * Vue18n does not support dot-notation in dict keys.
+         * As a workaround, we remove dots from message and from keys in dictionaries.
+         */
+        const dictKey = e.message.replaceAll('.', '');
```

Comment:
fix for the displaying change-password errors. For example, 'Wrong old password entered'